pkg/gcv/cf/cf_test.go

```diff
@@ -783,16 +770,32 @@ func TestCFAuditParsingWithRealAudit(t *testing.T) {
 			if err != nil {
 				t.Fatal(err)
 			}
-			for _, data := range tc.data {
-				cf.AddData(data)
+			result, reviewViolations := auditAndReview(t, cf, tc.data)
+
+			wantedConstraints := attachConstraints(tc.want, tc.constraints)
+			sortedViolations(wantedConstraints.Violations)
```

Comment:
Don't we want to store the sorted violations?

Reply:
Thanks, this mutates the array in place, so the name is actually not ideal here, fixed the naming on this.